scripts/lib/CIME/bless_test_results.py

```diff
@@ -59,7 +59,7 @@ def bless_history(test_name, case, baseline_name, baseline_root, report_only, fo
 def bless_test_results(baseline_name, baseline_root, test_root, compiler, test_id=None, namelists_only=False, hist_only=False,
                        report_only=False, force=False, bless_tests=None, no_skip_pass=False):
 ###############################################################################
-    test_id_glob = "*{}*".format(compiler) if test_id is None else "*{}*".format(test_id)
+    test_id_glob = "*{}[0-9][0-9]".format(compiler) if test_id is None else "*{}".format(test_id)
```

Comment:
I'm confused how this can work in the case where `test_id` is None. My test names often look like `SMS_D_Ld1_P4x1.f10_f10_musgs.I2000Clm50BgcCropQianRsGs.bishorn_gnu.clm-default.C.20191029_170705_zyp3pv`; it doesn't seem like a glob on '*gnu[0-9][0-9]' would pick that up. What am I missing?

Reply:
@billsacks is this not a test-id?  20191029_170705_zyp3pv

Reply:
oh - I see, it is a test id but it's not passed in.

Reply:
Why is this same assignment in two different places instead of in a shared location?   
@jgfouca I think that maybe you can find a better solution, these are tools that I don't use in my workflow but e3sm and cam developers do.